etc/config.pl

```diff
@@ -349,7 +349,7 @@
             google_analytics => 0,
             'inbox_max' => 2000,
             'interests' => 150,
-            'makepoll' => 0,
+            'makepoll' => 1,
```

Comment:
Always a good idea to put etc/config*.pl in your .gitignore! It means manually updating the configs if we add a new config variable, but it prevents having to be paranoid about what changes creep in.